python/src/aiconfig/Config.py

```diff
@@ -41,6 +42,10 @@
 ]
 for model in gpt_models_main:
     ModelParserRegistry.register_model_parser(DefaultOpenAIParser(model))
+
+ModelParserRegistry.register_model_parser(
+    OpenAIVisionParser("gpt-4-vision-preview")
+)
```

Comment:
cc @tanya-rai this would be the 3rd position from drop-down. If you want to change it, we can easily change in a future PR